pkg/bosh/converter/job_factory.go

```diff
@@ -152,51 +152,67 @@ func (f *JobFactory) VariableInterpolationJob(manifest bdm.Manifest) (*qjv1a1.Qu
 }
 
 // InstanceGroupManifestJob generates the job to create an instance group manifest
-func (f *JobFactory) InstanceGroupManifestJob(manifest bdm.Manifest) (*qjv1a1.QuarksJob, error) {
+func (f *JobFactory) InstanceGroupManifestJob(manifest bdm.Manifest, linkSecrets map[string]string) (*qjv1a1.QuarksJob, error) {
 	containers := []corev1.Container{}
 
 	// QuarksJob will always pick the latest version for versioned secrets
 	desiredManifestName := names.DesiredManifestName(manifest.Name, "1")
 
+	// Volume mounts for explicit linking provider links
```

Comment:
```suggestion
	// volumes for quarks links from native kube components
```

I tried to use words from the story.